docs/settings-gen/source/kibana-alert-action-settings.yml

```diff
@@ -4,13 +4,14 @@
 product: Kibana
 collection: Alerting and action settings in Kibana
 page_description: |
-  Alerting and actions are enabled by default in {kib}, but require you to configure the following:
-  . <<using-kibana-with-security,Set up {kib} to work with {stack} {security-features}>>.
-  . <<configuring-tls-kib-es,Set up TLS encryption between {kib} and {es}>>.
-  . If you are using an *on-premises* Elastic Stack deployment, <<general-alert-action-settings,specify a value for `xpack.encryptedSavedObjects.encryptionKey`>>.
+  Alerting and actions are enabled by default in {{kib}}, but require you to configure the following:
+  
+  1. [Set up {{kib}} to work with {{stack}} {{security-features}}](docs-content://deploy-manage/security/secure-your-cluster-deployment.md).
+  2. [Set up TLS encryption between {{kib}} and {{es}}](docs-content://deploy-manage/security/set-up-basic-security-plus-https.  md#encrypt-kibana-http).
+  3. If you are using an **on-premises** Elastic Stack deployment, [specify a value for `xpack.encryptedSavedObjects.encryptionKey`]  (#general-alert-action-settings).
```

Comment:
```suggestion
  3. If you are using an **on-premises** Elastic Stack deployment, [specify a value for `xpack.encryptedSavedObjects.encryptionKey`](#general-alert-action-settings).
```

Reply:
Nice catch! Fixed in [535452a](https://github.com/elastic/kibana/pull/214642/commits/535452adf6a9fa5d7fdd2d4c33cd302f784be8b9)